clients/python/llmengine/model.py

```diff
@@ -28,12 +29,15 @@ class Model(APIEngine):
     @assert_self_hosted
     def create(
```

Comment:
How do I specify the instance type? or are all models on the same instance (I need at least g5.12xl)